packages/backend/src/tests/propertyBased/arbitraries/arbitraryLimit.ts

```diff
@@ -0,0 +1,8 @@
+import { fc } from '@fast-check/vitest';
+
+export function arbitraryLimit(): fc.Arbitrary<number> {
+    return fc.integer({
+        min: 1,
+        max: 32767,
```

Comment:
how did you get to this number?

Also the min should actually be zero, not one